### ISSUE TITLE
Converting regular julia file to Notebook and vice versa

### ISSUE THREAD

**Issue by DonJayamanne** [julia-vscode/julia-vscode#2056]
Jupyter Notebooks supports exporting Notebooks to plain text (source) files.
The Jupyter extension in vs code supports this for Python as well.

Curious whether such a feature would be useful in Julia extension.
In Jupyter extension we found that there are plenty of users who prefer using plain Python files for experimentation (interactive computing), and then create a Notebook when they are happy with their results, I'm certain there are other reasons as well.

Thoughts?

**Comment by davidanthoff** [comment]
Yes, I think having various convert things would be very useful. Also for jmd files to notebooks. I think we'll essentially support .jl, .jmd and .ipynb file formats in the extension, and ideally we would have convert options in all directions.

**Comment by pfitzseb** [comment]
Yep, this should be fairly easy since we already have the notion of code cells in the extension.

**Comment by DonJayamanne** [comment]
Should we allow user to open the file as a notebook (have the same notebook look and feel but using raw julia file as the backing store) or should the julia file be converted into a notebook?

I can see benefits in both, but not sure which is better.

**Comment by pfitzseb** [comment]
Requiring conversion to a ipynb seems clearer to me.
